packages/@lwc/integration-karma/test/act/act-components/test-multiple-slots.js

```diff
@@ -27,9 +27,7 @@ export default function (define) {
                                 'ui-something',
                                 _uiSomething__default['default'],
                                 {
-                                    attrs: {
-                                        slot: 'first',
-                                    },
+                                    slotAssignment: 'first',
```

Comment:
We'll need to add an ACT compiler change for this

Reply:
This might be an issue if we decide to use API versioning because of the mechanics of how ACT compiler + API versioning works.

Try publishing an alpha version of LWC with these and run the jest tests on core with this change.